src/tasks.py

```diff
@@ -286,6 +286,42 @@ def get_depth_by_time_at_point(model_id: int, lat: float, lng: float) -> DepthTi
     return DepthTimePlot(depths, times)
 
 
+@app.task(base=OnFailureStateTask)
+def retrieve_medusa_input_parameters(scenario_id: int) -> Dict[str, Union[str, float]]:
+    """
+    Retrieve input parameters for the current scenario id. This is used for app file.
+
+    Parameters
+    ----------
+    scenario_id: int
+        The scenario ID of the pollution model run
+
+
+    Returns
+    -------
+    Dict[Point, int]
+        A dictionary contain information from Rainfall MEDUSA 2.0 database
+    """
+    # Get rainfall information
+    medusa_rainfall_event = retrieve_input_parameters(scenario_id)
+
+    # Write rainfall information into a dictionary
+    if medusa_rainfall_event is not None:
+
+        medusa_rainfall_dictionary = {
+            "antecedent_dry_days": float(medusa_rainfall_event['antecedent_dry_days']),
+            "average_rain_intensity": float(medusa_rainfall_event['average_rain_intensity']),
+            "event_duration": float(medusa_rainfall_event['event_duration']),
+            "rainfall_ph": float(medusa_rainfall_event['rainfall_ph']),
+            "geometry": str(medusa_rainfall_event['geometry'])
+        }
+
+    else:
+        medusa_rainfall_dictionary = None
+
+    return medusa_rainfall_dictionary
```

Comment:
```suggestion
    # Get rainfall scenario information
    return retrieve_input_parameters(scenario_id)
```
I think this may work, but try it out